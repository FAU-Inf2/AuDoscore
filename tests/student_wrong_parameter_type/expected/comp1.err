ERROR - Method public static long ToTest.toTest(long,long)[ToTest] does not match or does not exists in studentcode
Exception in thread "main" java.lang.Error
	at tools.ic.InterfaceComparer.main(InterfaceComparer.java:252)
make[1]: *** [run-comparer] Error 1
make: *** [compile-stage1] Error 2
