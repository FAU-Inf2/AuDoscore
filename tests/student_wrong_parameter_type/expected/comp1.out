rm -rf *.class
javac  -Xprefer:source -sourcepath interfaces student/*.java
javac  -Xprefer:source -sourcepath interfaces cleanroom/*.java
javac  -Xprefer:source -sourcepath interfaces:junit:student -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar junit/UnitTest.java
java -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar:junit:interfaces:student tester.tools.CheckAnnotation UnitTest
java -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar tester.tools.ForbiddenUseSearcher UnitTest > forbidden.out
if [ -s forbidden.out ]; then \
	cat forbidden.out 1>&2 ; \
	exit 1 ; \
fi
make run-comparer
make[1]: Verzeichnis „/audoscore/tests/student_wrong_parameter_type/test.402929“ wird betreten
java -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar tester.tools.InterfaceComparator UnitTest
make[1]: Verzeichnis „/audoscore/tests/student_wrong_parameter_type/test.402929“ wird verlassen
